es-vue-base/src/lib-mixins/forms.js

```diff
@@ -53,10 +53,19 @@ export default {
         * }
         */
         formErrors() {
-            if (!Object.prototype.hasOwnProperty.call(this.$v, 'form')) {
+            let hasFormAttribute = false;
+            try {
+                hasFormAttribute = Object.prototype.hasOwnProperty.call(this.$v, 'form');
+            } catch (e) {
+                return {};
+            }
```

Comment:
What's the catch here for? We should probably throw an warning instead of swallowing an error.

Reply:
This is basically catching `Object.prototype.hasOwnProperty.call(undefined)`. I suppose we could throw a warning, but I think this would only trigger when the validations section is omitted from the component object.

Reply:
Here's the context: https://github.com/EnergySage/es-ds/pull/550/files#r1096872668